libs/output-mapping/tests/Writer/StorageApiLocalTableWriterTest.php

```diff
@@ -939,14 +939,14 @@ public function testWriteMultipleErrors(): void
         } catch (InvalidOutputException $e) {
             self::assertStringContainsString(
                 'Failed to load table "' . self::OUTPUT_BUCKET . '.table10a": Some columns are ' .
-                'missing in the csv file. Missing columns: id,name. Expected columns: id,name. Please check if the ' .
```

Comment:
Z tohodle bude merzeníčko. Pokud bych to bral v rámci nových PLG věcí tak po nezdařeném importu by to ty nové sloupce mělo dropnout :( Možná založím nový task?

Reply:
Zajimavá situace, jako dávalo by to smysl jak říkáš

Reply:
Ale lituju toho kdo to bude implementovat :trollface: